pkgs/development/python-modules/cruft/default.nix

```
@@ -0,0 +1,65 @@
+{
+  buildPythonPackage,
+  cookiecutter,
+  fetchFromGitHub,
+  gitpython,
+  lib,
+  poetry-core,
+  pytestCheckHook,
+  pytest_7,
+  typer,
+}:
+buildPythonPackage rec {
+  pname = "cruft";
+  version = "2.15.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "cruft";
+    repo = "cruft";
+    rev = version;
+    hash = "sha256-qIVyNMoI3LsoOV/6XPa60Y1vTRvkezesF7wF9WVSLGk=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  nativeCheckInputs = [
+    (pytestCheckHook.override { pytest = pytest_7; })
+  ];
+
+  propagatedBuildInputs = [
+    cookiecutter
+    gitpython
+    typer
+  ];
```

```suggestion
    click
  ] ++ lib.optional (pythonOlder "3.11") toml;
```
If this package is a library, please add `toml` to support `python310Packages.cruft` and so on.
https://github.com/cruft/cruft/blob/c9f678a974484215273e8615ee2e121938febb72/pyproject.toml#L9-L17